packages/cli/src/modules/mcp/mcp.types.ts

```diff
@@ -33,6 +36,35 @@ export type SearchWorkflowsResult = {
 	count: number;
 };
 
-export type McpSettingsUpdateBody = {
-	mcpAccessEnabled: boolean;
+export type WorkflowDetailsNode = Omit<INode, 'credentials'>;
+
+/**
+ * Workflow object used in 'Get workflow details' tool
+ * We are deriving this from IWorkflowDB by omitting fields that are not surfaced through MCP
+ * and overriding types for some
+ */
+export type WorkflowDetailsWorkflow = Omit<
```

Comment:
Couldn't we infer this from the zod schema with [`z.infer`](https://zod.dev/basics#inferring-types)?